src/applications/simple-forms/form-upload/config/constants.js

```diff
@@ -190,3 +190,29 @@ export const ALERT_TOO_FEW_PAGES = (formNumber, pdfDownloadUrl, onCloseEvent) =>
       </React.Fragment>
     </VaAlert>,
   );
+
+export const ALERT_WRONG_FORM = (formNumber, pdfDownloadUrl, onCloseEvent) =>
+  Object.freeze(
+    <VaAlert
+      close-btn-aria-label="Close notification"
+      status="warning"
+      visible
+      closeable
+      onCloseEvent={onCloseEvent}
```

Comment:
I was curious about this implementation in the previous PR, but didn't leave a comment.  is this a one time event? or will it allow you do it more than once?